lib/openhab/core/configuration.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module OpenHAB
+  module Core
+    java_import org.openhab.core.config.core.Configuration
+
+    # Adds methods to core openHAB {org.openhab.core.config.core.Configuration} to make it more natural in Ruby
+    class Configuration
+      # inspect result is just the string representation
+      alias_method :inspect, :to_s
+
+      # Return the configuration value for the given key
+      # @!method [](key)
+      # @param [String] key
+      # @return [Object]
+      alias_method :[], :get
```

Comment:
where's `[]=`/`put`, `delete`/`remove`, and `keys`/`keySet`? for that matter, you might as well mix in `Enumerable`

Reply:
I wasn't sure whether to implement a full blown functionality, but I guess it's up to the caller to decide whether such operation is appropriate. I suppose they could still do it using the java methods anyway, so why not.